examples/quantization/brevitas/quantize_llm.py

```diff
@@ -57,8 +57,6 @@ def main(args):
     model = quantizer.model
 
     # Evaluation of the non-quantized model.
-    if use_accelerate:
```

Comment:
Removing this appears to have broken layerwise activations equalization. Specifically:

```bash
CUDA_VISIBLE_DEVICES=0 python quantize_llm.py --activations-equalization layerwise
```

Fails with multiple devices (inputs on CPU & weights on gpu). Either we need to:
 - `remove_hooks` + `offload_model` again before running activation equalization
 - Change how accelerate is called initially (call it explicitly, instead of implicitly when the model is instantiated through `from_pretrained`)
 
 I'm not sure why our `offload_model` method works differently to the one used when `from_pretrained` is called. @fxmarty, any thoughts on this?

Reply:
I can fix this I think :)